README.md

```diff
@@ -23,6 +24,10 @@ python3 home-alert.py -c /path/to/config/file
 ansible-playbook -i ./ansible/hosts ./ansible/playbook.yml
 ```
 
+Python modules needed:
+* Jinja2
+* pyyaml
```

Comment:
Is this documentation necessary? Ansible should transparently take care of installing all required python modules. I'd opt to remove this section since we actually need way more than just these two modules, and its likely to change in the future.

Reply:
My bad, should've clarified, you need these two modules to be able to run the home_alert.py script that creates the playbook.